precompile/modules/initia_stdlib/sources/dex.move

```diff
@@ -27,6 +27,20 @@ module initia_std::dex {
         coin_b_store: Object<FungibleStore>
     }
 
+    struct FlashSwap has key {
+        coin_a_borrow_amount: u64,
+        coin_b_borrow_amount: u64
+    }
+
+    /// FlashSwapLoan is following the hot potato pattern, so the borrower
+    /// need to pass the created FlashSwapLoan objec to this contract to 
+    /// destruct it.
+    ///
+    /// https://move-book.com/programmability/hot-potato-pattern.html
+    struct FlashSwapLoan {
+        pair_addr: address
```

Comment:
can we just add coin_a_amount and coin_b_amount to this hot potato struct?

Reply:
And also you need read interface maybe

Reply:
nvm, i thought we only need this hot potato, but i see your point. interface to query doesn't seem to be useful given the borrowed_amount are short-lived

Reply:
Added `get_borrow_amounts`